controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/noderepository/NodeRepositoryNode.java

```diff
@@ -76,6 +82,8 @@
     private NodeHistory[] history;
     @JsonProperty("allowedToBeDown")
     private Boolean allowedToBeDown;
+    @JsonProperty("suspendedSince")
```

Comment:
Now that I see it - this needs to be called suspendedSinceMillis.

Reply:
Fair enough.